presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -114,6 +115,12 @@ public enum StandardErrorCode
     QUERY_PLANNING_TIMEOUT(0x0001_0027, INTERNAL_ERROR),
     NATIVE_EXECUTION_TASK_ERROR(0x0001_0028, INTERNAL_ERROR),
     UNSUPPORTED_ANALYZER_TYPE(0x0001_0029, INTERNAL_ERROR),
+    CRYPT_ALGORITHM_ERROR(0x0001_002C, EXTERNAL),
```

Comment:
can you keep the error enums in the order of the error code

Reply:
Updated error codes in order